source/helpers/ApiRequest.js

```diff
@@ -19,7 +20,10 @@ const request = async (endpoint, method, data, headers, params) => {
   const token = await StorageService.getData(ACCESS_TOKEN_KEY);
   const { apiKey } =
     EnvironmentConfigurationService.getInstance().activeEndpoint;
-  const userAgent = `${name}/${version}/${Platform.OS}/${Platform.Version}`;
+
+  const friendlyVersion = getUserFriendlyAppVersion();
+
+  const userAgent = `${name}/${friendlyVersion}/${Platform.OS}/${Platform.Version}`;
```

Comment:
This change is creating an `UserAgent` that looks like the following:

`MittHelsingborg/v3.1-15-53025bb/ios/15.2`

Which means that we can no longer distinguish the current app version (in backend).
The **old** `UserAgent` on the other hand looked like the following:

`MittHelsingborg/1.1.6/ios/15.2`
where `1.1.6` is the current app version used.

So we either prevent this to be merged or come up with another solution regarding the `UserAgent` and specifically current application version.

Reply:
Change `UserAgent` to `MittHelsingborg/1.1.6/ios/15.2/v.3.1-15-53025bb` as agreed. By doing this change we won't interfere with current code in backend. We might want another `UserAgent` in the future, but that might need discussions first.